File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyListOfLabelRow.cs
/// <filename>
///     ListViews_ViewModelMyListOfLabelRow.cs
/// </filename>


namespace widgetassertions
{
    public class ListViews_ViewModelMyListOfLabelRow
  {
        private string MyValueLabelText;
     
        public     virtual string getMyValueLabelText()
    {
      return this.MyValueLabelText;
    }
     
        private bool isMyValueLabelEnabled;
     
        public     virtual bool getIsMyValueLabelEnabled()
    {
      return this.isMyValueLabelEnabled;
    }
     
        private bool isMyValueLabelVisible;
     
        public     virtual bool getIsMyValueLabelVisible()
    {
      return this.isMyValueLabelVisible;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyListOfLabelRow.cpp

#include "ListViews_ViewModelMyListOfLabelRow.h"
#include <string>

namespace widgetassertions
{
  std::string ListViews_ViewModelMyListOfLabelRow::getMyValueLabelText( ) 
  {
    return this->MyValueLabelText;
  }
  bool ListViews_ViewModelMyListOfLabelRow::getIsMyValueLabelEnabled( ) 
  {
    return this->isMyValueLabelEnabled;
  }
  bool ListViews_ViewModelMyListOfLabelRow::getIsMyValueLabelVisible( ) 
  {
    return this->isMyValueLabelVisible;
  }
  int ListViews_ViewModelMyListOfLabelRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

